.travis.yml

```diff
@@ -1,9 +1,24 @@
 # Copyright 2018 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 
-sudo: false
+dist: "trusty"
+
+install:
+  - yarn install
+  - pip3 install --user -r requirements.txt
+  - curl -sfL https://raw.githubusercontent.com/securego/gosec/master/install.sh | sh -s -- -b $GOPATH/bin 1.2.0
+
 language: node_js
 node_js:
-  - "8.3"
+  - "8"
```

Comment:
Martin's PR in https://github.com/vmware/precaution/pull/48/files is setting the minimum node version to 11.3.  So do we still need testing of 8 and 10?

Reply:
We don't "need" it but since all of them are currently supported releases I think we should try to remain compatible. Especially since we are not using any features from node 11.x as far as I know.

Reply:
Also the CI used to fail on 8.3 because ESLint requires 8.10 at minimum, just setting the major release number defaults to the latest release on that branch.